__tests__/api.test.js

```diff
@@ -353,3 +353,32 @@ describe("GET /api/articles (topic query)", () => {
       });
   });
 });
+
+describe("GET /api/articles/:article_id/comment_count", () => {
+  test("GET:200 responds with count of all comments for specific article", () => {
+    return request(app)
+      .get("/api/articles/1/comment_count")
+      .expect(200)
+      .then((res) => {
+        const count = res.body.count;
+        console.log(count);
+        expect(count).toBe(11);
+      });
+  });
+  test("GET:204 reponds with error message for article with no comments", () => {
```

Comment:
This test should just be testing that the comment_count value for an article with no comments is 0.

Reply:
It would also be a 200 status code.